Utilities/Hook.cs

```diff
@@ -0,0 +1,28 @@
+﻿using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SpecProj2.Utilities
+{
+    public class Hook : CommonDriver
+    {
+        
+        [Before]
```

Comment:
are these hooks correct, https://codebun.com/how-to-implement-hooks-in-specflow-c/ please use the correct specflow hooks